public/components/freelancer-create.js

```diff
@@ -81,14 +81,16 @@ class CreationForm extends React.Component {
               </label>
             </div>
 
-            <label className="category-selector">
-              Job category:
+            <div className="category-selector">
+              <span className="bar"></span>
+              <label>
+                Job Category
+              </label>
               <select ref="category">
                 <option value="selected" disabled>Please select a job category</option>
```

Comment:
This does not work, the form displays directly `testCategory1` and does not show the hint.
Also, the category selector looks a bit too large.